app/controllers/work_groups_controller.rb

```diff
@@ -16,11 +18,16 @@ def create
     return redirect_with_params('Please input at least 2 email addresses') if custom_params[:emails].split(' ').size < 2
 
     begin
-      users_and_work_groups = vars_for_mailer
-      users_and_work_groups[:users].each do |user|
-        DemoMailer.with(user: user, work_group: users_and_work_groups[:work_group]).invite.deliver_later
+      @users_and_work_groups = vars_for_mailer
```

Comment:
not sure why we use an instance variable here, can you please change it to a regular one?